moped-editor/src/views/staff/EditStaffView.js

```diff
@@ -25,8 +25,7 @@ const validationSchema = yup.object().shape({
   first_name: yup.string().required(),
   last_name: yup.string().required(),
   title: yup.string().required(),
-  workgroup: yup.string().required(),
-  workgroup_id: yup.string().required(),
+  workgroup_id: yup.number().integer().required(),
```

Comment:
Like Patrick said the other day, "it is the little things." Really happy to see this improvement. 🙌